edx_notifications/server/web/static/edx_notifications/js/vendor/dev/backbone.js

```diff
@@ -0,0 +1,1608 @@
+//     Backbone.js 1.1.2
```

Comment:
I moved those out of edx_notifications/server/web and into the testserver. The assumption will be that the hosting application (like the LMS) will provide these files (but we need it in our testserver).